lms/djangoapps/grades/tasks.py

```diff
@@ -23,48 +23,37 @@
 
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
-def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible):
+def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible, score_deleted):
```

Comment:
We're going to have to avoid breaking prod workers when we deploy this. The problem arises when we have new edx-platform code running on the lms, queuing tasks with a `score_deleted` parameter that will be picked up by workers running the previous version of edx-platform, not expecting that parameter. This causes a spike in error rates [that looks like this](https://rpm.newrelic.com/accounts/88178/applications/5476895/traced_errors?tw%5Bend%5D=1479327300&tw%5Bstart%5D=1479325500)

I think we'll either have to shim this task, or change the task name slightly.